tests/suites/test_suite_ecp.data

```diff
@@ -312,6 +312,96 @@ genkey_mx_known_answer:447:"ffffffffffffffffffffffffffffffffffffffffffffffffffff
 ECP generate Montgomery key: Curve448, not enough entropy
 genkey_mx_known_answer:447:"4f0102030405060708090a0b0c0d0e0f101112131415161718191a1b1c1d1e1f202122232425262728292a2b2c2d2e2f30313233343536":""
 
+ECP generate in range: 4
+genkey_sw_many:"04":1000
+
+ECP generate in range: 5
+genkey_sw_many:"05":1000
+
+ECP generate in range: 6
+genkey_sw_many:"06":1000
+
+ECP generate in range: 7
+genkey_sw_many:"07":1000
+
+ECP generate in range: 8
+genkey_sw_many:"08":1000
+
+ECP generate in range: 9
+genkey_sw_many:"09":1000
+
+ECP generate in range: 10
+genkey_sw_many:"0a":1000
+
+ECP generate in range: 11
+genkey_sw_many:"0b":1000
+
+ECP generate in range: 12
+genkey_sw_many:"0c":1000
+
+ECP generate in range: 255
+genkey_sw_many:"ff":100
+
+ECP generate in range: 256
+genkey_sw_many:"0100":100
+
+ECP generate in range: 257
+genkey_sw_many:"0101":100
+
+ECP generate in range: 272
+genkey_sw_many:"0110":100
+
+ECP generate in range: 2^64-1
+genkey_sw_many:"ffffffffffffffff":100
+
+ECP generate in range: 2^64
+genkey_sw_many:"010000000000000000":100
+
+ECP generate in range: 2^64+1
+genkey_sw_many:"010000000000000001":100
+
+ECP generate in range: 2^64+2^63
+genkey_sw_many:"018000000000000000":100
+
+ECP generate in range: 2^65-1
+genkey_sw_many:"01ffffffffffffffff":100
+
+ECP generate in range: 2^65
+genkey_sw_many:"020000000000000000":100
+
+ECP generate in range: 2^65+1
+genkey_sw_many:"020000000000000001":100
+
+ECP generate in range: 2^65+2^64
+genkey_sw_many:"030000000000000000":100
+
+ECP generate in range: 2^66+2^65
+genkey_sw_many:"060000000000000000":100
+
+ECP generate in range: 2^71-1
+genkey_sw_many:"7fffffffffffffffff":100
+
+ECP generate in range: 2^71
+genkey_sw_many:"800000000000000000":100
+
+ECP generate in range: 2^71+1
+genkey_sw_many:"800000000000000001":100
+
+ECP generate in range: 2^71+2^63
```

Comment:
Out of curiosity, why these values? I can see the idea with 2^64+2^63 but here it's not obvious to me. (Not saying there's anything to change here, just curious.)